### ISSUE TITLE
[Feature request] Account analytics across databases

### ISSUE THREAD

**Issue by techhazard** [keeweb/keeweb#610]
I sometimes have old copies of my password database open at the same time as my current version, as well as other databases altogether. I would like to use keeweb to help me manage multiple databases. What I'm looking for is one or more of the following:
- compare databases: show duplicates and differences
- combine databases: merge exact duplicates and ask for conflict resolution for non-exact 
duplicates
- password analysis: flag multiple occurences of the same username/password and email/password, as well as weak passwords in general

 Would that be good features to have?  
Is it doable?

~- what was wrong?~
- app version: 1.4.1
- your user-agent (from Settings/Help section): desktop  

~- does it happen on Demo or New database?~
~- if you are using a test db without your personal data, please attach it~
~- please, open dev tools in your browser and attach output log from Console tab ~
~    (if you are using desktop app, devtools can be opened from Settings/General/Advanced)~

**Comment by antelle** [comment]
Db comparison and merge should be done as a plugin, it's not that frequently used feature.
Password analysis is audit, #49, will appear in future releases.

**Comment by techhazard** [comment]
Are there any resources available for developing plugins? Are there any existing plugins already? I've only used vanilla KeeWeb so far 😅

**Comment by antelle** [comment]
Only this: https://github.com/keeweb/keeweb/wiki/Plugins
Plugins are very new to keeweb, they will appear in the next version.

**Comment by techhazard** [comment]
That's why I've never heard of that feature before 😄. Thanks, I'll look into it.

**Comment by thomasfrobieter** [comment]
+1 for the merge feature. I sync the database file with Nextloud and got a conflict all couple of month. I've merged it with the regular Keepass Windows App in the past. Sadly my JS skills are not good enough to solve this as a plugin -_-

Please consider updating the issue title, kinda hard to find this issue, maybe: "[Feature request] Database analysis, comparing & merging".

**Comment by lordvalium** [comment]
Agree, that would be nice. Also a double password should be emphasized.

**Comment by techhazard** [comment]
I am no longer using Keeweb, I wish everyone good luck with this :slightly_smiling_face:

**Comment by newhinton** [comment]
I think it could be helpful if the "tab"-mechanism of keeweb could be overhauled. 
If someone has two files open and unlocked, transfering passwords between those two files is very hard as soon as more than a single field has to be transfered.

The reason for that is that each time the Database is switched, the "old" view gets discarded, and when the user returns to the first database, he has to search again for the last entry, and repeat for the new database. I generally think that displaying different databases in the same view can be confusing. It would be good if it was possible to have different "views" for each database, however i see that this may be a very subjective opinion. (also, i am not able to judge how hard it would be to actually implement this, but i can imagine that may be a lot of work, too much to actually do it)

**Comment by leolivier** [comment]
Hi @antelle 
I don't know if database merging is a frequently used feature or not but I also need it 😃 
Actually, I think that storing password dbs on webdav/cloud systems triggers synchronization issues now and then and you discover one month later that on your 2 different devices, you are not using the same file anymore...
I wrote the haveIbeenPwned plugins a while ago so I know a bit of that but now I'm engaged in other projects so I can't look at this unfortunately 😞

**Comment by ANaPhOrAX** [comment]
any update on this? would be a really useful feature

**Comment by leolivier** [comment]
I'm afraid keeweb is dead, there has been no release since 4 years... Or maybe there's a maintained fork somewhere?
